js/.jscsrc

```diff
@@ -5,6 +5,7 @@
   "disallowMultipleLineStrings": true,
   "disallowMultipleVarDecl": true,
   "disallowQuotedKeysInObjects": "allButReserved",
+  //"disallowSemicolons": true,
```

Comment:
I sort of left it on purpose as a placeholder since we are "supposed" to not use semicolons. But `/grunt/*` is using them.

I'll just remove it completely, I guess.